jobsdb/jobsdb_test.go

```diff
@@ -36,251 +33,196 @@ import (
 	. "github.com/rudderlabs/rudder-server/utils/tx" //nolint:staticcheck
 )
 
-var _ = Describe("Calculate newDSIdx for internal migrations", Ordered, func() {
-	BeforeAll(func() {
+func TestComputeInsertIdx(t *testing.T) {
```

Comment:
God bless you Siddarth :joy: (I don't like Ginkgo :sweat:)

Reply:
same😅